messaging-experiments.yaml

```diff
@@ -200,3 +200,118 @@
                   label: Start Browsing
                   action:
                     navigate: true
+- id: MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST
+  enabled: true
+  # The filter `!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)` is used to exclude
+  # users who have switched channels or re-installed the browser without having a new profile
+  targeting: env.version >= '81.' && env.channel == 'default' && localeLanguageCode == "en" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'zero-stage-test' in activeExperiments)
+  arguments:
+    slug: zero-stage-test
+    active: true
+    features: []
+    startDate: ""
+    endDate: ""
+    proposedDuration: 0
+    proposedEnrollment: 0
+    referenceBranch: ""
+    bucketConfig:
+      count: 9000
+      start: 0
+      total: 10000
+      namespace: zero-stage-test
+      randomizationUnit: normandy_id
+    userFacingName: "Multi-Stage About:Welcome Zero Stage"
+    userFacingDescription: Test of the Zero Stage JSON flag, with and without override content
```

Comment:
Should update description to "Study to validate with a minimal onboarding experience."